src/devtools/client/debugger/src/actions/breakpoints/modify.ts

```diff
@@ -103,10 +103,14 @@ export function addBreakpoint(
     if (!column) {
       const lineBreakpoints = getPossibleBreakpointsForSource(getState(), sourceId);
       if (!lineBreakpoints) {
-        console.debug(`No possible breakpoints on line ${line} of source ${sourceId}`);
+        console.debug(`No possible breakpoints for source ${sourceId}`);
         return;
       }
       column = Math.min(...lineBreakpoints.filter(bp => bp.line === line).map(bp => bp.column));
+      if (!column || column === Infinity) {
```

Comment:
I think this caused a regression (see FE-828)

`column === 0` will fail this

Reply:
Yep, shit, sorry, thanks Brian!

Reply:
No worries! 😆 This one gets me all the time.